app/parse.py

```diff
@@ -8,8 +13,51 @@
     tags: list[str]
 
 
-def main(output_csv_path: str) -> None:
-    pass
+def parse_single_quote(quote: Tag) -> Quote:
+    return Quote(
+        text=quote.select_one(".text").text,
+        author=quote.select_one(".author").text,
+        tags=[tag.get_text() for tag in quote.select(".tag")],
+    )
+
+
+def get_quotes_from_page(url: str) -> [Quote]:
+    response = requests.get(url)
+    response.raise_for_status()
+    soup = BeautifulSoup(response.text, "html.parser")
+
+    quotes = [parse_single_quote(quote) for quote in soup.select(".quote")]
+
+    next_button = soup.select_one(".next > a")
+    next_page_url = f"{BASE_URL}{next_button["href"]}" if next_button else None
+
+    return quotes, next_page_url
+
+
+def get_quotes_in_page() -> list[Quote]:
```

Comment:
The function name `get_quotes_in_page` is misleading. It actually retrieves quotes from all pages, not just a single page. Consider renaming it to `get_all_quotes` for clarity.